fieldmanager.php

```diff
@@ -499,6 +499,23 @@ function fm_sanitize_textarea( $value ) {
 }
 
 /**
+ * Call wp_die() with an error message for Fieldmanager_Field::_failed_validation().
+ *
+ * @since 1.1.0
+ *
+ * @param string $error_message Optional. Error message in addition to the
+ *                              default message for validation errors.
+ */
+function fm_failed_validation_to_wp_die( $error_message = '' ) {
+	wp_die( esc_html(
+		$error_message
+		. "\n\n"
+		. __( "You may be able to use your browser's back button to resolve this error.", 'fieldmanager' )
+	) );
+}
+add_action( 'fm_failed_validation', 'fm_failed_validation_to_wp_die' );
```

Comment:
I'm not entirely sold on this, even though it successfully maintains backwards compatibility. The problems with dying are:

1. This field's data may be lost. One could argue that's OK, because it's invalid.
2. Fields hooked to save before this field (by Fieldmanager or not) **will** save.
3. Fields hooked to save after this field (by Fieldmanager or not) **will not** save and their data may be lost.

I'm at a bit of a loss on how best to handle this instead. I'm inclined to say that we should not save the invalid field by default, but make that filtered so one could elect to still save the invalid data (to prevent data loss). Then let all other fields save and do one of two things:

1. Hook into code later in execution (prior to the redirect) to die.
2. Let the request and redirect succeed, then on the next page render an admin notice with the error message. We could do this by setting a transient for the user with the error message, and filter the redirect to set a query arg to indicate that there was an error and that FM should look for it. Upon loading the next screen, if the query arg is present that there was an error, FM would load the transient and output the error message.

Of these ideas, I lean toward (2). However, I'd really love a better option.